tests/app/test_model.py

```diff
@@ -223,6 +223,47 @@ def test_letter_notification_serializes_with_subject(client, sample_letter_templ
     assert res["subject"] == "Template subject"
 
 
+def test_notification_serialize_with_cost_info_for_sms(client, sample_template, sms_rate):
+    notification = create_notification(sample_template, billable_units=2)
+
+    response = notification.serialize_with_cost_info()
+
+    assert response["is_cost_data_ready"] is True
+    assert response["cost_details"] == {"sms_fragments": 2, "rate_multiplier": 1, "rate": "0.0227"}
```

Comment:
I haven't looked into what's behind the difference, but when trying this out locally I got the value of rate_multiplier as `1.0` instead of `1`

Reply:
Ahh, yes, I saw this, too, thanks for pointing out that it's not consistent - I will make it consistent 👍🏼

Reply:
I added a commit for this, with my reasoning in the commit message.